components/Avatar/Avatar.tsx

```diff
@@ -213,31 +229,32 @@ function Avatar(props: AvatarProps): JSX.Element {
     [`--border-style`]: borderColor ? "solid" : "none",
   };
 
+  const $avatarImage = (
```

Comment:
why the `$` ?

Reply:
good call on reducing duplication here

Reply:
The $ is just a convention I like to use to specify that the variable itself is JSX